conf/agent.conf.template

```diff
@@ -155,6 +155,31 @@ cygnusagent.sinks.mysql-sink.batch_size = 100
 cygunsagent.sinks.mysql-sink.batch_timeout = 30
 
 # ============================================
+# OrionPostgreSQLSink configuration
+# channel name from where to read notification events
+cygnusagent.sinks.postgresql-sink.channel = postgresql-channel
+# sink class, must not be changed
+cygnusagent.sinks.postgresql-sink.type = com.telefonica.iot.cygnus.sinks.OrionPostgreSQLSink
+# true if the grouping feature is enabled for this sink, false otherwise
+cygnusagent.sinks.postgresql-sink.enable_grouping = false
+# the FQDN/IP address where the PostgreSQL server runs
+cygnusagent.sinks.postgresql-sink.postgresql_host = x.y.z.w
+# the port where the PostgreSQL server listens for incomming connections
+cygnusagent.sinks.postgresql-sink.postgresql_port = 5432
+# a valid user in the PostgreSQL server
+cygnusagent.sinks.postgresql-sink.postgresql_username = root
+# password for the user above
+cygnusagent.sinks.postgresql-sink.postgresql_password = xxxxxxxxxxxxx
+# how the attributes are stored, either per row either per column (row, column)
+cygnusagent.sinks.postgresql-sink.attr_persistence = column
+# select the table type from table-by-destination and table-by-service-path
+cygnusagent.sinks.postgresql-sink.table_type = table-by-destination
```

Comment:
In the meantime, we have added a general `data_model` parameter that is configured at `OrionSink` level. This new parameter may have values `dm-by-service`, `dm-by-service-path`, `dm-by-entity` and `dm-by-attribute`. `dm-by-service-path` matches `table-by-service-path`,  and `dm-by-entity` matches `table-by-destination`. Thus, this `table_type` parameter must be replaced by `data_model`.